locales/templates/app.pot

```diff
@@ -219,11 +231,8 @@ msgstr ""
 #. content to show up in the "recent apps" screen      and prevents the user
 #. from taking screenshots or casting the app to other displays. %1$s will be
 #. replaced with the      name of the app (e.g. Firefox Focus).
-#, c-format
 msgctxt "preference_privacy_secure_mode_summary"
-msgid ""
-"Turn off stealth mode to take screenshots and see sites you’ve visited "
-"using %1$s in your Recent Apps list"
+msgid "Hide webpages when switching apps"
```

Comment:
It was decided to change the description completely (The setting is still the same). User testing revealed that the previous explanation was too complex and users didn't understand it. So the UX team decided to replace it with a new simpler explanation. Should we create a new key for it and remove the old one?